mne/viz/backends/_pyvista.py

```diff
@@ -1345,7 +1346,13 @@ def _is_mesa(plotter):
                     "surface rendering, consider upgrading to 18.3.6 or "
                     "later."
                 )
-    return is_mesa
+        renderer = re.findall(
+            r"OpenGL renderer string:  ([0-9.a-zA-Z\-])+",
```

Comment:
are the two spaces after the colon intentional here?

Reply:
Yep! But I'll change it to ` +` because it seems cleaner / more robust